src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentRepository.php

```diff
@@ -41,6 +41,19 @@ public function findOneByOrderId($paymentId, $orderId): ?PaymentInterface
         ;
     }
 
+    public function findOneByOrderToken(string $paymentId, string $orderToken): ?PaymentInterface
+    {
+        return $this->createQueryBuilder('p')
```

Comment:
As this undocumented best practice, we are using `o` for default query builder
```suggestion
        return $this->createQueryBuilder('o')
```